src/ucp/core/ucp_ep.c

```diff
@@ -788,7 +787,7 @@ void ucp_ep_disconnected(ucp_ep_h ep, int force)
     ucp_ep_destroy_internal(ep);
 }
 
-static unsigned ucp_ep_do_disconnect(void *arg)
+unsigned ucp_ep_local_disconnect_progress(void *arg)
```

Comment:
pls keep old name ucp_ep_do_disconnect

Reply:
why? it's used as progress callback as well

Reply:
https://github.com/openucx/ucx/pull/4445#discussion_r349912367